SPECS/cmake/cmake.spec

```diff
@@ -90,6 +90,9 @@ bin/ctest --force-new-ctest-process --rerun-failed --output-on-failure
 %{_libdir}/rpm/macros.d/macros.cmake
 
 %changelog
+* Wed Jun 19 2024 CBL-Mariner Servicing Account <cblmargh@microsoft.com> - 3.29.6-1
```

Comment:
nit. can you write your name in the changelog? 
Gen rule was if you did changes to the Auto Patch/Auto Upgrade PRs you had to write your name.